CorePreferences/src/au/gov/asd/tac/constellation/preferences/GraphOptionsPanel.java

```diff
@@ -64,138 +76,115 @@ public int getBlazeOpacity() {
     // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
     private void initComponents() {
 
-        blazeSizePanel = new javax.swing.JPanel();
-        blazeSlider = new javax.swing.JSlider();
-        BlazeSizeDescription = new javax.swing.JLabel();
-        blazeOpacitySlider = new javax.swing.JSlider();
-        BlazeSizeDescription1 = new javax.swing.JLabel();
-        BlazeColourPanel = new javax.swing.JPanel();
-        BlazeColourDescription = new javax.swing.JLabel();
-        BlazeColourPlaceholder = new javax.swing.JLabel();
-        notePanel = new javax.swing.JPanel();
-        noteLabel = new javax.swing.JLabel();
+        blazeSizePanel = new JPanel();
+        blazeSlider = new JSlider();
+        BlazeSizeDescription = new JLabel();
+        blazeOpacitySlider = new JSlider();
+        BlazeSizeDescription1 = new JLabel();
```

Comment:
General refactoring:
**BlazeSizeDescription1** - upon further inspection was added by me, it can be renamed to **BlazeOpacityDescription**
**blazeSlider** - Can be renamed to **BlazeSizeSlider** to match BlazeOpacitySlider

Also, I seem to have used camel case and title case - Whichever is preferred I am happy to have them changed to be consistent with the rest of the declarations.

Reply:
@aldebaran30701 I've updated the variables and its now ready for a review.